ee/tabby-db/migrations/0031_doc-crawler-urls.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE doc_crawler_urls;
```

Comment:
```suggestion
DROP TABLE web_crawler_urls;
```

Please make the name consistent cross the PR